save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestCreationComponent.kt

```diff
@@ -98,7 +99,23 @@ fun ChildrenBuilder.showContestCreationModal(
     }
 }
 
-private fun String.dateToLocalDateTime(time: LocalTime = LocalTime(0, 0, 0)) = LocalDateTime(LocalDate.parse(this), time)
+private fun String.dateStringToLocalDateTime(time: LocalTime = LocalTime(0, 0, 0)) = LocalDateTime(
+    LocalDate.parse(this),
+    time,
+)
+
+/**
+ * @param startTime
+ * @param endTime
+ */
+fun isDateRangeValid(startTime: LocalDateTime?, endTime: LocalDateTime?) = if (startTime != null && endTime != null) {
+    startTime < endTime
+} else {
+    true
+}
+
+private fun isButtonDisabled(contestDto: ContestDto) = contestDto.endTime == null || contestDto.startTime == null || !isDateRangeValid(contestDto.startTime, contestDto.endTime) ||
```

Comment:
may be move `startTime == null` and `endTime == null` to `isDateRangeValid`?

Reply:
The point is that we do want button to be disabled when date is not picked, but we also do not want to be shown as invalid